mapdataprefetcher/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Aug 02 16:14:57 EEST 2021
+distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```

Comment:
Should be 
distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip